src/data/nat/totient.lean

```diff
@@ -10,6 +10,7 @@ open_locale big_operators
 
 namespace nat
 
+/-- Euler's totient function -/
```

Comment:
Anyone who understands this doc string will know what the function does from its name, can you expand it?